README.md

```diff
@@ -212,6 +212,23 @@ Further, there's a new call named `gds.graph.get` (`graphdatascience` only) whic
 The idea is to have a way of creating `Graph`s for already projected graphs, without having to do a new projection.
 
 
+### Pipeline catalog utils
+
+All procedures from the [GDS Pipeline catalog](https://neo4j.com/docs/graph-data-science/2.0-preview/pipeline-catalog/) are supported with `graphdatascience`.
+Some examples are (where `pipe` is a machine learning training pipeline object):
+
+```python
+res = gds.beta.pipeline.list()
+assert len(res) == 1  # Exactly one pipeline is in the catalog
+
+res = gds.beta.pipeline.drop(pipe)
+assert res["pipelineName"] == pipe.name()
+```
+
+Further, there's a new call named `gds.pipeline.get` (`graphdatascience` only) which takes a pipeline name as input and returns a training pipeline object if a pipeline of that name exists in the user's pipeline catalog.
```

Comment:
This is the readme, which doesn't have a contextual dimension of time. Maybe we should avoid saying "new" here?

Reply:
Good point :)

Reply:
Done